src/Image/Image.tsx

```diff
@@ -37,7 +37,7 @@ export const Image: React.FunctionComponent<ImageProps> = React.memo(
                 {props.useImgTag && (
                     <img
                         id={props.id}
-                        className={"img-tag" + (props.className ? ` ${props.className}` : "")}
+                        className={`img-tag${props.onClick ? " link" : ""} ${props.className ? props.className : ""}`}
```

Comment:
This will leave an empty space if className is not passed. Here's how to fix it
```suggestion
                        className={`img-tag${props.onClick ? " link" : ""}${props.className ? ` ${props.className}` : ""}`}
```